docs/source/certificates/certificates.md

```diff
@@ -0,0 +1,173 @@
+## Certificate Management Guide
+
+**Audience**: Fabric network admins
+
+This guide provides a brief overview, resources, and technical details for Fabric network administrators to manage certificates (certs).
+
+### Key Concepts
+
+**Registration** – A registration is a username (enrollment ID) and password stored in the certificate authority (CA). This is created by a CA admin level user and has no expiration. The registration also contains any roles and attributes required.
+
+**Enrollment** – Based on the registration username and password, an enrollment (or identity), can be created. The enrollment contains a public/private key pair and the (encoded) roles and attributes from the registration. The public key in the enrollment is referred to as a certificate, which is a signed certificate that is required to use Hyperledger Fabric.
```

Comment:
I'm okay with a bit of simplification since the audience doesn't necessarily have a security background, but I'd still try to be factually correct. A public key is _not_ a certificate. The former is a key, the latter is the key, a description of the subject and the signature of an issuer on key and description.

Reply:
...contains a public/private key pair and an x509 certificate issued by the organization's certificate authority. The certificate encodes roles and attributes from the registration.

Wdyt?